.github/workflows/ci.yml

```diff
@@ -117,3 +117,15 @@ jobs:
           yarn --frozen-lockfile
           /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
           yarn run test
+  verify_indexing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Index Top 100 Ruby gems
+        run: bundle exec rake index:topgems
```

Comment:
This should only run if the lockfile or the `ruby_indexer` dir have been modified. There's no need to run in other PRs.

Reply:
Yes! Let's do that an run the action on any commit or PR that modifies the lockfile or ruby_indexer.